packages/yarnpkg-core/sources/Workspace.ts

```diff
@@ -69,7 +69,7 @@ export class Workspace {
         expandDirectories: false,
         onlyDirectories: true,
         onlyFiles: false,
-        ignore: [`node_modules`, `.git`],
+        ignore: [`**/node_modules/**`, `**/.git/**`],
```

Comment:
As mentioned by @arcanis here https://github.com/yarnpkg/berry/pull/1480#discussion_r442846807, we should ignore the `.yarn` folder as well

Reply:
@merceyz Thanks! Added `**/.yarn/**` as well.